htmlify.js

```diff
@@ -1,20 +1,41 @@
-// setup mutation observer, because this is the new cool way to see if elements are mounted
 let observer;
 observer = new MutationObserver((mutations) => {
   mutations.forEach((mutation) => {
-    const messageElement = document.querySelector(
-      ".RunDetails-Description .message"
-    );
+    // Convert any description on the **build page** from text to html
+    const buildPageDescription = document.querySelector(".RunDetails-Description .message");
+    convertTextToHtml(buildPageDescription);
 
-    // we need to know if there's a tag character, in the text... if there isn't, don't go crazy
-    const messageElementContainsTag = messageElement?.innerText?.includes("<");
-    if (messageElement && messageElementContainsTag) {
-      // this is the real magic - set the html of the message box to whatever the text of the message box is
-      messageElement.innerHTML = messageElement.innerText;
-    }
+    // Convert any description on the **project page** from text to html
+    const messageColumn = document.querySelectorAll(".JTable-cell-contents .RunMessageCell");
+    
+    messageColumn.forEach(messageCell => {
+      // The title is set to raw HTML as well so clean that up
+      if (messageCell.title) {
+        messageCell.title = "";
+      }
+      const messageCellInnerDescription = messageCell.querySelector(".RunMessageCellInner .unstyled-link");
+      convertTextToHtml(messageCellInnerDescription);
+    });
   });
 });
 
+/**
+ * Turns the raw HTML text rendered by an element to actual HTML if the raw text contains HTML elements
+ * @param {Element} element HTML element  
+ */
+function convertTextToHtml(element) {
+  // Presence of "<" might indicate that it might be raw HTML
+  if (element?.innerText?.includes("<")) {
+    // Test if the text is indeed HTML by using an HTML element like div
+    const tempDiv = document.createElement("div");
+    tempDiv.innerHTML = element.innerText;
+    // If the innerText is rendered differently, highly likely that it contains valid HTML elements
+    if (tempDiv.innerText != element.innerText) {
+      element.innerHTML = element.innerText;
+    }
+  }
+}
+
```

Comment:
Can this block be moved to the top (so that it's more obvious what's being used in the mutation observer)?

Reply:
Sure, why not.